### PR TITLE
Fix 417 / corrected HTML tags in dots component for proper HTML validation

### PR DESCRIPTION
**Pull request by humbak**
Fixes: #417

### CODE REVIEW

File: src/components/CarouselDots.js [hunk 1]
```diff
@@ -17,14 +17,14 @@ class CarouselDots extends Component {
     const numberOfSlides = this.props.number || this.props.thumbnails.length;
     const moduloItem = this.calculateButtonValue() % numberOfSlides;
 
-    return this.props.onChange(this.props.value - ( moduloItem - index));
+    return this.props.onChange(this.props.value - (moduloItem - index));
```
Comment (piotr-s-brainhub) [inline src/components/CarouselDots.js:20]: could you configure lint to fail in such a situation?

File: src/components/CarouselDots.js [hunk 2]
```diff
@@ -17,14 +17,14 @@ class CarouselDots extends Component {
     const numberOfSlides = this.props.number || this.props.thumbnails.length;
```
Comment (piotr-s-brainhub) [inline src/components/CarouselDots.js:17]: you have a typo in the branch name:

`fix/corrected-sematic-of-html-tags-in-dots-component`

`sematic` -> `semantic`

File: src/components/CarouselDots.js [hunk 3]
```diff
@@ -17,14 +17,14 @@ class CarouselDots extends Component {
...
-    return this.props.onChange(this.props.value - ( moduloItem - index));
+    return this.props.onChange(this.props.value - (moduloItem - index));
   };
 
   calculateButtonValue = () => {
     const numberOfSlides = this.props.number || this.props.thumbnails.length;
     return this.props.value >= 0
       ? this.props.value
-      : this.props.value + numberOfSlides * Math.ceil(Math.abs(this.props.value/numberOfSlides));
+      : this.props.value + numberOfSlides * Math.ceil(Math.abs(this.props.value / numberOfSlides));
```
Comment (piotr-s-brainhub) [inline src/components/CarouselDots.js:27]: lint should fail also in this case

File: src/components/CarouselDots.js (hunk 2 above)
Comment (humbak) [reply src/components/CarouselDots.js:17]: Is any chance to correct it during the review?

File: src/components/CarouselDots.js (hunk 1 above)
Comment (humbak) [reply src/components/CarouselDots.js:20]: We may move this comment to a separate ticket. I did it as a The Boy Scout Rule.

File: src/components/CarouselDots.js (hunk 3 above)
Comment (humbak) [reply src/components/CarouselDots.js:27]: as above

File: src/components/CarouselDots.js (hunk 1 above)
Comment (piotr-s-brainhub) [reply src/components/CarouselDots.js:20]: OK so you can create a ticket and paste its link here

File: src/components/CarouselDots.js (hunk 2 above)
Comment (piotr-s-brainhub) [reply src/components/CarouselDots.js:17]: I guess you need to open a new PR

File: src/components/CarouselDots.js (hunk 1 above)
Comment (humbak) [reply src/components/CarouselDots.js:20]: https://github.com/adam-golab/eslint-config-brainhub/pull/23

**Pull request closed without merging**